README.md

```diff
@@ -40,48 +40,31 @@ comprehensive documentation, visit:
 
 (You do not need to do this if you downloaded a pre-built package)
 
-### Building with Ant
-
-Lucene and Solr are built using [Apache Ant](http://ant.apache.org/).  To build
-Lucene and Solr, run:
-
-`ant compile`
-
-If you see an error about Ivy missing while invoking Ant (e.g., `.ant/lib does
-not exist`), run `ant ivy-bootstrap` and retry.
-
-Sometimes you may face issues with Ivy (e.g., an incompletely downloaded artifact).
-Cleaning up the Ivy cache and retrying is a workaround for most of such issues: 
-
-`rm -rf ~/.ivy2/cache`
-
-The Solr server can then be packaged and prepared for startup by running the
-following command from the `solr/` directory:
-
-`ant server`
 
 ### Building with Gradle
 
-There is ongoing work (see [LUCENE-9077](https://issues.apache.org/jira/browse/LUCENE-9077))
-to switch the legacy ant-based build system to [gradle](https://gradle.org/).
-Please give it a try!
-
-At the moment of writing, the gradle build requires precisely Java 11 
-(it may or may not work with newer Java versions).
+As of 9.0, Lucene/Solr uses [Gradle](https://gradle.org/) as the build
+system. Installatcion instructions [here](https://gradle.org/install/).
```

Comment:
s/Installatcion/Installation

Do we need to install this at all? I thought we use the wrapper.